kafka.go

```diff
@@ -457,6 +457,20 @@ func (client *KafkaClient) PublishAuditLog(auditLogBuilder *AuditLogBuilder) err
 	return nil
 }
 
+func (client *KafkaClient) PublishMessage(topic, messageKey string, message interface{}, errorCallback PublishErrorCallbackFunc) error {
+	msgBytes, marshalErr := json.Marshal(&message)
```

Comment:
Can we use `err` for error variable here? Seems unnecessary to use specific err name here since the err variable is just this one (except the one we returned below which has no variable either).